calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.0-release-notes.mdx

```diff
@@ -52,4 +52,12 @@ For more information, see [Configure egress traffic to multiple external network
 This includes the following:
 * Non-root context whenever possible and added drop ALL `Capabilities`
 * Root context and privilege escalation are used only when necessary
-* `SeccompProfile` is set to `RuntimeDefault`
\ No newline at end of file
+* `SeccompProfile` is set to `RuntimeDefault`
+
+## Known issues
+
+* When using persistent storage for Anomaly Detection, currently only `tigera-anomaly-detection` can be used as a value for `StorageClassName`.
```

Comment:
These are the release notes for v3.15.1:
```
Known issues
L7 log collection fails to deploy on CIS hardened clusters. As a result, some cards in the manager UI dashboard will not display any metrics.
Upgrading Calico Enterprise v3.15.1 on Rancher/RKE from Calico Enterprise v3.13.0 currently requires manually terminating the calico-node container for upgrade to proceed.
```

I think you copied the one from v3.15.0

Reply:
Fixed. Issues taken from 3.15.1 and adjusted for new version.